site/community/release_guide.md

```diff
@@ -303,7 +303,7 @@ Use Maven release plugin to stage these artifacts on the Apache Nexus repository
     mvn release:perform [-DdryRun] [-Darguments="-Dmaven.javadoc.skip=true -DskipTests=true"] [-Dresume=true]
 
 > If `release:perform` failed, 
-> delete the release tag: git tag -d ${VERSION} && git push apache :refs/tags/${VERSION}
+> delete the release tag: git tag -d release-${VERSION} && git push apache :refs/tags/release-${VERSION}
```

Comment:
${TAG} is `release-${VERSION}`. you can change it to `git tag -d ${TAG} && git push apache :refs/tags/${TAG}`.